src/arrayop.d

```diff
@@ -27,7 +27,9 @@ import ddmd.visitor;
 /**************************************
  * Hash table of array op functions already generated or known about.
  */
-extern (C++) __gshared AA* arrayfuncs;
+//extern (C++) __gshared AA* arrayfuncs;
+__gshared FuncDeclaration[void*] arrayfuncs;
```

Comment:
Why commenting it instead of just removing it ?
Also, could it be made `private` ?